### ISSUE TITLE
CSS Rule not working on Validation Message

### ISSUE THREAD

**Issue by vsfeedback** [dotnet/aspnetcore#30705]
_This issue has been moved from [a ticket on Developer Community](https://developercommunity.visualstudio.com/t/CSS-Rule-not-working-on-Validation-Messa/1351391)._

---
Eureka! - I'm learning Blazor and while I was doing my first form, I noticed that something was not working with CSS isolation.  What I was trying to do is to style my error messages (change font color, size, etc)... so as a practice, I decided to create some CSS that only applies to my current page.  My next step was to create a CSS file called the same as my component, (please view my attached screenshot), then create my rules inside.

I did notice that if I change the style for an existing h3, it works... but if I create a style rule for the div that holds the error, it doesn't do anything, as if the CSS doesn't exist... and if the same rule is written in the same page, it works as expected.

In other words, it read some rules and ignore others, maybe an error? any help will be appreciated, thank you so much for your prompt attention.

---
#### Feedback Bot on 3/1/2021, 01:06 AM: 

<p>We have directed your feedback to the appropriate engineering team for further evaluation. The team will review the feedback and notify you about the next steps.</p>

---
### Original Solutions
(no solutions)

**Comment by mkArtakMSFT** [comment]
Thank you for filing this issue. In order for us to investigate this issue, please provide a minimalistic repro project (ideally a GitHub repo) that illustrates the problem.
